devicetypes/drzwave/ezmultipli.src/ezmultipli.groovy

```diff
@@ -108,8 +108,7 @@ metadata {
 			input "TempMin", "number", title: "Temperature Report Frequency", description: "Temperature report sent every N minutes [0-127]", range: "0..127", defaultValue: 6, displayDuringSetup: true, required: false
 		}
 		section {
-			input title: "Temperature Offset", description: "This feature allows you to correct any temperature variations by selecting an offset. Ex: If your sensor consistently reports a temp that's 5 degrees too warm, you'd enter '-5'. If 3 degrees too cold, enter '+3'.", displayDuringSetup: true, type: "paragraph", element: "paragraph"
-			input "TempAdj", "number", title: "Degrees", description: "Adjust temperature up/down N tenths of a degree F [(-127)-(+128)]", range: "-127..128", defaultValue: 0, displayDuringSetup: true, required: false
+			input "TempAdj", "number", title: "Temperature Offset", description: "Adjust temperature by this many degrees", range: "-127..128", defaultValue: 0, displayDuringSetup: true, required: false
```

Comment:
all the other ones use `tempOffset` instead of `TempAdj` do we want to make this match?

Reply:
I thought about it but that change would mean I'd have to also run a script to call `updated` on all existing devices which didn't seem worth the effort.

Reply:
Actually, this one might be worth holding off on. At least, I'd probably remove the description as that seems like it is not accurate for what the device preference actually is (I might have added that and not fully updated it). It is possible that the offset here would need to be converted if we were switching this to degrees, since it sounds like the preference is for tenths of a degree... (Unless you already investigated this and it checks out.)

Reply:
Good catch. Will fix.

Reply:
Pushed a fix in ac4442611d2f599819f8e082f7edaa4973788a12